syft/frameworks/torch/hook/hook.py

```diff
@@ -828,6 +828,26 @@ def is_wrapper(self, it_is_a_wrapper):
 
         tensor_type.native_grad_fn = tensor_type.grad_fn
 
+        # Added support for the torch.size() command
+        def size(self, dim=None):
+            """Hook the size to return the shape with a callable syntax"""
+            if dim is None:
+                return self.shape
+            return self.shape[dim]
+
+        torch.Tensor.native_size = torch.Tensor.size
+        torch.Tensor.size = size
+
+        def flip_hook_native_size(self):
+            """
```

Comment:
Thanks for re-using my hack, but actually I'm very unhappy with this hack, I don't think we should use it. I would either try to modify the PyTorch implement to have them use shape instead of size, or try another technique

Reply:
So should we discard this fix for .size() completely?

Reply:
Yes, because we can't modify the hook at runtime it will be far too slow and bad practice.

Reply:
Removed the flip_hook_native_size